src/scripts/main.js

```diff
@@ -2,7 +2,27 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const advancedDB = [ ...people ];
 
-// write your code here
+for (let i = 0; i < advancedDB.length; i++) {
+  advancedDB[i] = {
+    ...advancedDB[i],
+    age: advancedDB[i].died - advancedDB[i].born,
+    century: Math.ceil(advancedDB[i].died / 100),
+  };
+}
+
+const table = document.querySelector('.dashboard>tbody');
+
+for (const row of advancedDB) {
+  table.insertAdjacentHTML('beforeend',
+    `<tr>
+      <td>${row.name}</td>
+      <td>${row.sex === 'f' ? 'Female' : 'Male'}</td>
+      <td>${row.born}</td>
+      <td>${row.died}</td>
+      <td>${row.age}</td>
+      <td>${row.century <= 20 ? row.century + 'th' : row.century + 'st'}</td>
+    </tr>`
```

Comment:
```suggestion
const table = document.querySelector('.dashboard>tbody');

for (const row of people) {
  table.insertAdjacentHTML('beforeend',
    `<tr>
      <td>${row.name}</td>
      <td>${row.sex === 'f' ? 'Female' : 'Male'}</td>
      <td>${row.born}</td>
      <td>${row.died}</td>
      <td>${row.died - row.born}</td>
      <td>${Math.ceil(row.died / 100)}</td>
    </tr>`
```

Why did you create a copy of the array? You can take information right from the 'people' and put I directly to your code. Don't you think it looks more understandable and shorter?
Also, there's no need to add endings ('th' or 'st'). But I appreciate you've made it. Nice